apps/web/src/validators/process/coreRequirements.ts

```diff
@@ -9,7 +9,7 @@ export function hasAllEmptyFields({
   Pool,
   "language" | "securityClearance" | "location" | "isRemote"
 >): boolean {
-  const hasLocation = isRemote || (location?.en && location.fr);
+  const hasLocation = isRemote ?? (location?.en && location.fr);
```

Comment:
I think this should be using ||.  Coalescing doesn't make as much sense here.  Same with L25.

Reply:
I took your suggestion to ignore boolean primitives.

[ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)